sdk/servicebus/service-bus/src/receivers/shared.ts

```diff
@@ -226,3 +226,8 @@ function settleMessage(
     throw translateServiceBusError(err);
   });
 }
+
+export function numberOfEmptyIncomingSlots(receiver: any): number {
+  const incomingDeliveries = (receiver as any)?.session?._session?.incoming?.deliveries;
```

Comment:
Are we creating a PR in rhea to expose session on the receiver?

Reply:
Yes, on the way!